recipes/libelfin/all/patches/946dde57e5caef8297b9339f3a7971401d540840.patch

```diff
@@ -0,0 +1,217 @@
+diff --git a/dwarf/Makefile b/dwarf/Makefile
+index 71456bd..2d47003 100644
+--- a/dwarf/Makefile
++++ b/dwarf/Makefile
```

Comment:
I guess you can remove the `Makefile` hunks from the patch, as you use your own `CMakeLists.txt`.

Reply:
Done